python/trace-python.c

```diff
@@ -1017,6 +1022,13 @@ static void __attribute__((destructor)) uftrace_trace_python_finish(void)
 	free(main_dir);
 }
 
+static PyObject *uftrace_trace_python_exit(PyObject *self, PyObject *obj)
+{
+	int n = PyLong_AsLong(obj);
+	uftrace_trace_python_finish();
+	_exit(n);
```

Comment:
Maybe not necessary, but to prevent possible compiler complaint, return NULL here.